lang/interpret_test/TestAstFunc1/metaparams0.graph

```diff
@@ -25,7 +25,6 @@ Vertex: bool(true)
 Vertex: bool(true)
 Vertex: bool(true)
 Vertex: bool(true)
-Vertex: bool(true)
```

Comment:
Looks right here! I trust you figured out how the test stuff works? FWIW I don't write these .graph files manually, I look at the output from running the test with -v and if it seems plausible I use that.

Reply:
Yeah exactly I took your advice and did just that.